crates/ui/src/dock/state.rs

```diff
@@ -65,6 +68,16 @@ pub struct DockItemState {
     pub info: DockItemInfo,
 }
 
+#[derive(Debug, Clone, Serialize, Deserialize, PartialEq)]
+pub struct TilePanelState {
+    pub(crate) panel_state: DockItemState,
+    pub(crate) x: Pixels,
+    pub(crate) y: Pixels,
+    pub(crate) w: Pixels,
+    pub(crate) h: Pixels,
```

Comment:
`bounds: Bounds<Pixels>`

Reply:
GPUI's Bounds, Point how is serialisable

https://github.com/zed-industries/zed/pull/21783